### DISCUSSION TITLE
Instructional videos on R-Instat

### DISCUSSION THREAD

**Discussion by rdstern** [IDEMSInternational/R-Instat#7153] [Ideas]
I suggest, following the review, this becomes a good time to prepare a series of instructional videos on R-Instat.  I also find, that quite naturally many of the possible videos correspond the the evaluation criteria used in the recent R-Instat review.  So, we might consciously have his criteria as a possible default title where appropriate.
The first 3 I have in mind are
a) `Simple data entry`.  I think that's "Data Editor", or one aspect of it.  I think we can have a better title there, but I would cover the entry of the small data set described in #7130 here.  Some recent issues suggest improvements that relate to this topic.  So it may become the first video in the series (after the tutorial videos), but may not be the first we produce.
b) `Import Data File Types` is one of his categories and we score well here.  I was thinking of a video combining import and export, but let's do them separately.  
c) So we need another video on the topic `"Export data file types"`.  That's another strength of R-Instat and recognised in the review.
d) I still want one on `Factors and Labels in R-Instat`.  I suggest we are really good on that, and we also have a very out-of-date tutorial 3 on that topic.  That doesn't correspond to a particular item.
e) I am really keen on `Import data from the library.`  That's where we discuss data.  That could usefully split into 2 videos, with the R side and the Instat side considered separately.  Then I would like a third, small additional video on importing from corpora which is done differently.  It has interesting datasets - now described in the help, but we need our halfway dialogues to import.  That's a nice opportunity to combine interesting data with the idea that using an R-command can be made pretty easy.  Copying, adapting and tweaking R commands is much easier than writing a whole script!
f) `Multiple output windows`.  We score 0 there, but that is now available.  So a video on the new output windows would be good.  I would not have put that on the list before the review, but it fits well as a topic.
g) `Hides R code by default` is a topic we can now discuss.  We score 0, and can now change that, with the new output windows.  Worth a short video, to discuss the topic.
h) Variable Metadata view.  Nice one to do.
i)  Statistics Distributions.  We score well here.  We have a few more to add, but don't necessarily have to wait, before a nice video, on the distributions and also random sampling from them.  We could complement that by sampling types of data, which we also do well - the wakefield package!  Perhaps that could be an additional to this?
j) We score very highly on the review's data wrangling.  Great to use that as a title, and describe the prepare menu overall.  Almost certainly this will lead to a number of additional, more particular videos on aspects of this topic.
k) `Variable filtering`.  We have a great filtering system.
l) `Variable selection`.  That's a partner and not a category yet.  (Perhaps he may add it in the future?  It is called sets in SPSS.) We are aiming to cope with wide datasets and this is one important tool.  
m) `Variable search in dialogues`. We (correctly) score 0 here, but will add facilities this year.  
n) `Pub-quality tables w/out R code steps` is now in Ease of Use in the review.  We are working on that topic.
   
And so on!  Exciting!  I must get on with some scripts.

**Comment by berylwaswa** [comment]
This is indeed exciting! We look forward to starting to work on the videos as soon as possible. We are well poised to work on more videos. However, we will need the desktops in the Maseno office. Please advise on the way forward and we can start the work as soon as you can produce the first script.
